src/FSharp.Configuration/ProvidedTypes.fs

```diff
@@ -941,7 +942,9 @@ type ProvidedSymbolType(kind: SymbolKind, args: Type list) =
         | SymbolKind.Array _ -> typeof<System.Array>
         | SymbolKind.Pointer -> typeof<System.ValueType>
         | SymbolKind.ByRef -> typeof<System.ValueType>
-        | SymbolKind.Generic gty  -> ProvidedSymbolType.convType args gty.BaseType
+        | SymbolKind.Generic gty  -> 
+            if gty.BaseType = null then null else
+            ProvidedSymbolType.convType args gty.BaseType
```

Comment:
We should not change `ProvidedTypes.fs/fsi`. We copy them from FSharp.TypeProviders.StarterPack repo. If you find a bug, please make a PR to that repo as well.